xbmc/guilib/GraphicContext.h

```diff
@@ -296,6 +296,6 @@ class CGraphicContext : public CCriticalSection,
  \brief
  */
 
-XBMC_GLOBAL(CGraphicContext,g_graphicsContext);
-
+XBMC_GLOBAL_REF(CGraphicContext,g_graphicsContext);
+#define g_graphicsContext XBMC_GLOBAL_USE(CGraphicContext)
```

Comment:
This was added as a "temporary" fix to the BSS space initialization order problem. The goal is to get rid of the globals all together but simply removing these references will destabilize the build and it will be different on each platform. It may work on some and not on others.

It's "crap" because the use of globals is crap. If you're stuck with the globals then it fixes a major issue that was causing major stability issues when I first joined the project.

Please read the docs here: https://github.com/xbmc/xbmc/blob/master/xbmc/utils/GlobalsHandling.h#L25